rend3/shaders/lighting/pixel.glsl

```diff
@@ -47,7 +47,11 @@ PixelData get_per_pixel_data(MATERIAL_TYPE material) {
 
     if (MATERIAL_FLAG(FLAGS_ALBEDO_ACTIVE)) {
         if (HAS_ALBEDO_TEXTURE) {
-            pixel.albedo = texture(sampler2D(ALBEDO_TEXTURE, linear_sampler), coords);
+            if (MATERIAL_FLAG(FLAGS_UNLIT)) {
```

Comment:
On second thought we should just have a new material flag that determines which of the two samplers to use.